spec.bs

```diff
@@ -1340,6 +1342,53 @@ Note: This is because we need to ensure that we do not leak <var ignore>creator<
 document's referrer|referrer=], [=Document/origin=], [=creator base url=], [=Document/policy
 container=], across the fenced frame boundary.
 
+<h3 id=local-policy-container-monkeypatch>Local requests policy container patch</h3>
+
+When making [=local scheme=] requests, <{iframe}>s inherit their [=Document/policy container=] from
```

Comment:
This reads as if you're targeting subresource requests, but I believe this only applies to navigation requests, right? Hence all the changes to "navigation params" stuff below?

Reply:
I've updated this to say "clone their [policy container](https://html.spec.whatwg.org/multipage/dom.html#concept-document-policy-container) from the [policy container](https://html.spec.whatwg.org/multipage/dom.html#concept-document-policy-container) of the [Document](https://dom.spec.whatwg.org/#document) initiating the [navigation request](https://fetch.spec.whatwg.org/#navigation-request)". This matches what's happening in [determine navigation params policy container](https://html.spec.whatwg.org/multipage/browsers.html#determining-navigation-params-policy-container) with the _initiatorPolicyContainer_ piece.